.github/scripts/update_disabled_tests.py

```diff
@@ -57,9 +69,13 @@ def get_disable_issues() -> Dict[Any, Any]:
 
 
 def validate_and_sort(issues_json: Dict[str, Any]) -> None:
-    assert issues_json["total_count"] == len(issues_json["items"]), f"# issues {len(issues_json['items'])} does not" \
+    assert issues_json["total_count"] == len(issues_json["items"]), (
+        f"# issues {len(issues_json['items'])} does not"
```

Comment:
minor nit: Might the error message be clearer with something like: "Number of issues does not equal total_count. Received X, while total_count is Y"

Reply:
Sure, this change comes from ufmt linter, but I can change the message accordingly to save some efforts later on